_sass/overrides.sass

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
1. Isn't `scss` usually the extension / file path, not `sass`? I'm fine with it either way.
2. What about `_includes/scss/` as the directory instead of another top level? (I actually have js and scss files in `assets/js` and `assets/css` in one of my projects, not sure what the latest conventions are)

Reply:
We could use either. `scss` looks more like `css`, but the directory is named `sass` so I went with that.

I'm using the `_sass/overrides.sass` because that's what Just the Docs sugguests for [integrating custom class styles](https://pmarsceill.github.io/just-the-docs/docs/customization/#override-styles). They have a top level `_sass` directory so we're kind of stuck with it. It just didn't show up before because we always pulled the default at build time.

Reply:
Ah, I see. Aren't these just defining new values though? If they aren't overriding values then they don't need to respect that naming. (Again, not a strong opinion, but good to discuss for if/when we add more here)

Reply:
I think that moving the styles to `_includes/scss` would be the right thing to do once we start adding more custom styles. That would probably involve adding `scss_files` to doc front matters like `js_files`, and would require figuring out how to get Jekyll to compile `scss -> css` and get things playing nicely with Just the Docs' styling by making sure ours are applied after all of theirs.

For now I moved `_sass/overrides.sass` to `_sass/custom/custom.scss`. Their docs imply that they intended for that file to be used for users to customize their predefined `scss` variables. That's why I had the styles in `overrides`, but I think `custom` makes more sense regardless.